IlmBase/IlmThread/IlmThreadMutex.h

```diff
@@ -116,7 +120,7 @@ class ILMTHREAD_EXPORT Mutex
     void	lock () const;
     void	unlock () const;
 
-    #if defined _WIN32 || defined _WIN64
+    #if (defined _WIN32 || defined _WIN64)
```

Comment:
tbh I didn't know #if defined FOO works without parens. we should do it one way consistently.

Reply:
Yeah, I wanted to unify style but missed a few.